samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -362,10 +480,14 @@ int main(void)
 			}
 		}
 
+#if defined(CONFIG_USB_DEVICE_AUDIO)
+		/* Never stop streaming when using USB Audio as input */
+		k_sleep(K_FOREVER);
+#else
```

Comment:
What if the cable gets unplugged? 

Can also be `IS_ENABLED` instead

Reply:
I'll have to check on the USB events - but yes, should be handled.